integration-tests/package.json

```diff
@@ -7,8 +7,8 @@
   "license": "MIT",
   "scripts": {
     "lint": "yarn lint:fix && yarn lint:check",
-    "lint:fix": "prettier --config ./.prettierrc.json --write 'test/**/*.ts'",
-    "lint:check": "eslint -c .eslintrc.js --ext .ts --format stylish .",
+    "lint:fix": "yarn lint:check --fix",
+    "lint:check": "eslint -c .eslintrc.js --ext .ts  .",
```

Comment:
I think in `specs` it isn't unreasonable to keep using `prettier` since it's being used for Markdown linting as opposed to JS or TS and it would likely be overkill to use ESLint for that purpose